### PR TITLE
[twitter] New update_status action.

### PR DESCRIPTION
**Pull request by Kami**

### CODE REVIEW

**Comment by Kami** [comment]
Just FYI - keys on the screenshot are revoked.

File: packs/twitter/actions/update_status.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+  name: "update_status"
+  runner_type: "run-python"
+  description: "Update your status (post a new tweet)."
+  enabled: true
+  entry_point: "update_status.py"
+  parameters:
+    status:
+      type: "string"
```
Comment (lakshmi-kannan) [inline packs/twitter/actions/update_status.yaml]: UTF-8 supported?

**Comment by lakshmi-kannan** [comment]
LGTM

File: packs/twitter/actions/update_status.yaml (hunk 1 above)
Comment (Kami) [inline packs/twitter/actions/update_status.yaml]: Yep, Twitter supports unicode, but I believe jsonschema only has "string" type and Python json schema parser we use handles unicode just fine, right?

Comment (lakshmi-kannan) [inline packs/twitter/actions/update_status.yaml]: That is my question, dear sir! Can you please validate?

Comment (Kami) [inline packs/twitter/actions/update_status.yaml]: Yep, looks like it does:

* https://github.com/Julian/jsonschema/blob/master/jsonschema/validators.py#L52
* https://github.com/Julian/jsonschema/blob/master/jsonschema/compat.py#L30

**Pull request merged**